config/config-mailgun.js

```diff
@@ -0,0 +1,23 @@
+const gunMail = require("mailgun.js");
+
+require("dotenv").config();
+
+gunMail.setApiKey=process.env.MAILGUN_API_KEY;
+
+const sendVerificationEmail = async (user) => {
```

Comment:
Warto by było uczynić funkcję odpowiedzialną za wysyłanie wiadomości bardziej generyczną. To pozwoli na jej łatwiejsze wykorzystanie w różnych częściach aplikacji, a także ułatwi modyfikację i rozbudowę w przyszłości.
Spróbuj dodać parametry do funkcji, takie jak adresat, temat, treść wiadomości i inne opcje konfiguracyjne, które mogą być potrzebne w zależności od kontekstu użycia. Dzięki temu będziesz mogła używać tej samej funkcji do wysyłania różnych rodzajów wiadomości bez konieczności tworzenia nowej funkcji za każdym razem, gdy potrzebujesz wysłać coś innego.